tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -205,6 +205,10 @@ def create_parser(product_choices=None):
                               help="Allow the wptrunner to install fonts on your system")
     config_group.add_argument("--font-dir", action="store", type=abs_path, dest="font_dir",
                               help="Path to local font installation directory", default=None)
+    config_group.add_argument("--headless", action="store_true",
+                              help="Run browser in headless mode", default=None)
+    config_group.add_argument("--no-headless", action="store_false", dest="headless",
+                              help="Don't run browser in headless mode")
```

Comment:
Can we clarify what the default behaviour is in the help text?

Reply:
Weirdly, not easily. Because`wpt run` sets its own per-browser default. So we'd have to override the help text in that case.